.travis.yml

```diff
@@ -3,9 +3,13 @@ language: php
 php:
   - 5.5
 
+before_install:
+  - mysql -e "create database IF NOT EXISTS dashboardhub;" -uroot
```

Comment:
would it not be safer to not set this up as root and create a db user and assign?